### PR TITLE
lazygamereviews.KEmulator version 1.0.3

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106133)

### CODE REVIEW

File: manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: lazygamereviews.KEmulator
+PackageVersion: 1.0.3
+PackageLocale: en-US
+Publisher: lazygamereviews
+PackageName: KEmulator
```
Comment (stephengillie) [inline manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://archive.org/details/@lazygamereviews
PackageName: KEmulator
PackageUrl: https://archive.org/details/KEmulator103
```

Comment (upintheairsheep) [reply manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml:9]: I believe it is not a violation

**Comment by stephengillie** [comment]
This appears to be a Java application. It includes many EXEs, including `java.exe` and `KEmulator.jar`, but no `KEmulator.exe`. Attempting to launch `KEmulator.jar` with the bundled `java.exe` gives: 
> Error: Could not find or load main class C:\Users\User\AppData\Local\Temp\WinGet\lazygamereviews.KEmulator.1.0.3\extracted\KEmulator.jar

File: manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml:9]: > Usually, Archive.org would count as a 3rd party site, but here it appears to be the publisher's only upload location. I do not know if this violates Archive.org's TOS, and if so this package might need to be removed.

"Access to the Archive’s Collections is provided at no cost to you and is granted for scholarship and research purposes only."

File: manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: lazygamereviews.KEmulator
+PackageVersion: 1.0.3
+Installers:
+- InstallerUrl: https://archive.org/download/KEmulator103/KEmulator%201.0.3%20modded.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (Trenly) [inline manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
> This appears to be a Java application. It includes many EXEs, including `java.exe` and `KEmulator.jar`, but no `KEmulator.exe`. Attempting to launch `KEmulator.jar` with the bundled `java.exe` gives:
> 
> > Error: Could not find or load main class C:\Users\User\AppData\Local\Temp\WinGet\lazygamereviews.KEmulator.1.0.3\extracted\KEmulator.jar

I was able to see KEmulator.exe. However, it does seem to rely on the linked DLLs

image

File: manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml:9]: @stephengillie / @denelon - The more I think about this, should the Waived-Validation-Domain be removed in light of the ToS for archive.org?

Comment (stephengillie) [reply manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.locale.en-US.yaml:9]: > > Usually, Archive.org would count as a 3rd party site, but here it appears to be the publisher's only upload location. I do not know if this violates Archive.org's TOS, and if so this package might need to be removed.
> 
> "Access to the Archive’s Collections is provided at no cost to you and is granted for scholarship and research purposes only."
> 
> https://archive.org/about/terms.php

Thanks for finding this.

File: manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: lazygamereviews.KEmulator
+PackageVersion: 1.0.3
+Installers:
+- InstallerUrl: https://archive.org/download/KEmulator103/KEmulator%201.0.3%20modded.zip
```
Comment (Trenly) [inline manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.installer.yaml:7]: Archive.org's TOS only allow for scholarship and research purposes only. Winget does not fall into the category of Scholarship and/or Research, so we cannot use them as a source

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
Marking as blocking issue so this doesn’t get closed

[Policy] Zip Binary

**Comment by Trenly** [comment]
@upintheairsheep - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**